### PR TITLE
NF: UnburyType is an enum

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Upstream unit test tests various Unbury. Looking at them, I thought using an enum would be more clear than using string

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
 import android.text.style.StyleSpan;
 
 import com.ichi2.anki.R;
+import com.ichi2.anki.exception.ManuallyReportedException;
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java:34]: This doesn't seem necessary

Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java:34]: I don't know how or why android studio decided to add it

Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java:34]: Corrected

**Comment by mikehardy** [comment]
I do love strongly typed systems - thanks @Arthur-Milchior

**Pull request merged**